go/runtime/registry/config/config.go

```diff
@@ -0,0 +1,222 @@
+// Package config implements global configuration options.
+package config
+
+import (
+	"fmt"
+	"time"
+)
+
+// RuntimeMode is the runtime mode.
+type RuntimeMode string
```

Comment:
This should now be removed in favor of the top-level node mode which already covers all the needed options.